examples/amp-story/ads/app-install.html

```diff
@@ -8,9 +8,9 @@
   <script async custom-element="amp-fit-text" src="https://cdn.ampproject.org/v0/amp-fit-text-0.1.js"></script>
 
   <meta name="amp-cta-type" content="INSTALL">
-  <meta name="amp-cta-url" content="https://www.example.com">
+  <meta name="amp-cta-url" content="https://www.amp.dev">
   <meta name="amp4ads-vars-attribution-icon" content="https://tpc.googlesyndication.com/pagead/images/adchoices/icon.png">
```

Comment:
Maybe we should substitute it with the icon that is local: /test/fixtures/e2e/amphtml-ads/resource/icon.png

Reply:
Sure, done.